rfcs/2020-11-02-remap-infallible-functions.md

```diff
@@ -0,0 +1,371 @@
+# RFC … - 2020-11-02 - Remap Language Fallible and Infallible Functions
+
+By introducing a distinction between fallible and infallible functions, we can
```

Comment:
The terminology here is a bit confusing to me. I would assume that "fallible" means a function that is able to fail (e.g. JSON parsing), and "infallible" means a function that cannot fail (e.g. `now()`).